server/controllers/controllerCenters.js

```diff
@@ -36,7 +36,58 @@ class Center {
       });
   }
 
-  
+  /**
+       * edit center
+       * @param {object} req expres req object
+       * @param {object} res exp res object
+       * @returns {json} json
+       * @memberof CenterController
+       */
+  editCenter(req, res) {
+    const { id } = req.params;
+    let editCenter;
+
+    db.centers.forEach((center) => {
+      if (center.id === parseInt(id, 10)) {
```

Comment:
you keep repeating a particular block of when you could actaully destructure, read more about object destructuring this seems too verbosed.